procedures/katello/branch.md.erb

```diff
@@ -1,43 +1,83 @@
 * [ ] Create KATELLO-<%= release %> branches
   * [ ] [katello](https://github.com/katello/katello)
   * [ ] [katello-installer](https://github.com/katello/katello-installer)
 * [ ] Bump versions to <%= develop %>-master
   * [ ] [katello](https://github.com/katello/katello)
   * [ ] [foreman-installer](https://github.com/katello/katello-installer)
+* [ ] Branch docs
+  * Use `./tools branch-docs config/katello/2.1.yaml` to branch automatically or by hand:
+    * [ ] Copy [nightly manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/plugins/katello/nightly) to <%= release %>
+    * [ ] Create sidebar from [nightly](https://github.com/theforeman/theforeman.org/tree/gh-pages/_includes/plugins/katello/sidebar_nightly.html) to `sidebar_<%= release %>.html`
+    * [ ] Add <%= release %> to [layout](https://github.com/theforeman/theforeman.org/blob/gh-pages/_layouts/plugins/katello/documentation.html#L3) and set `display` to include `-RC`
+  * [ ] Draft release notes in markdown ([example](https://gist.github.com/domcleal/5567450)), with these sections (and do not use personal pronouns):
+    * [ ] Headline features: half a dozen important features with a few sentences description each
+    * [ ] Upgrade notes: all important notices that users must be aware of before upgrading
+    * [ ] Release notes: bullet point list by category of most changes, excluding bug fixes for issues introduced during the release cycle, include link to bug numbers. You can auto-generate changes using the 'changelog' command in [tool_belt](https://github.com/theforeman/tool_belt)
+* [ ] Update API docs and open a PR with the changes:
 
-# Website
+      VERSION=<%= short_version %>
+      $GITDIR=~/git  #directory containing theforeman.org git repo
 
-* [ ] Copy [nightly manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/plugins/katello/nightly) to <%= release %>
-* [ ] Create sidebar from [nightly](https://github.com/theforeman/theforeman.org/tree/gh-pages/_includes/plugins/katello/sidebar_nightly.html) to `sidebar_<%= release %>.html`
-* [ ] Add <%= release %> to [layout](https://github.com/theforeman/theforeman.org/blob/gh-pages/_layouts/plugins/katello/documentation.html#L3) and set `display` to include `-RC`
-* [ ] Draft release notes in markdown ([example](https://gist.github.com/domcleal/5567450)), with these sections (and do not use personal pronouns):
-  * [ ] Headline features: half a dozen important features with a few sentences description each
-  * [ ] Upgrade notes: all important notices that users must be aware of before upgrading
-  * [ ] Release notes: bullet point list by category of most changes, excluding bug fixes for issues introduced during the release cycle, include link to bug numbers. You can auto-generate changes using the 'changelog' command in [tool_belt](https://github.com/theforeman/tool_belt)
+      FOREMAN_APIPIE_LANGS=en rake apipie:cache
```

Comment:
In Foreman we did release with all languages. I wonder if we should standardize

Reply:
Release or put on the website? Or both?

Reply:
Website. https://theforeman.org/api/1.20/ has all languages for example, but we haven't translated any other content.

Reply:
I'd happily accept a PR with a script in https://github.com/theforeman/theforeman.org/tree/gh-pages/scripts the other way around: point it to a GITDIR with a katello checkout. From there we can easily modify it to also handle the foreman updating process.